### PR TITLE
WD-11513 Changed occurance of "ROCKs" to "rocks" in ceph

### PR DESCRIPTION
**Pull request by immortalcodes**
## Done
- Changed occurance of "ROCKs" to "rocks"

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- Please find [Demo](https://ubuntu-com-13888.demos.haus/ceph).
- Also the [copydoc](https://docs.google.com/document/d/1gpEjTcdJor2yTGfMz7T_xIfs7BtV6rbBPFYfNZFe52Y/edit) for the change.

## Issue / Card
[WD-11513](https://warthogs.atlassian.net/browse/WD-11513)

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-11513]: https://warthogs.atlassian.net/browse/WD-11513?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by carkod** [comment]
Can you provide a link to the copy docs and to the demo, so it's easier for people outside of Sites to review?

File: templates/ceph/index.html [hunk 1]
```diff
@@ -305,6 +306,6 @@ <h3 class="p-heading--4">Managed Ceph</h3>
   </div>
 </section>
 <section class="p-strip--light is-deep is-bordered">
-  {% include "shared/_resources_ceph.html"%}
+  { % include "shared/_resources_ceph.html"%}
```
Comment (carkod) [inline templates/ceph/index.html:309]: Is this by Djlint? Normally there should not be a space between `{` and `%` and there should be a space between `.html"` and `%`

**Review by carkod — COMMENTED** [review]
LGTM, just a minor comment.

Also maybe you can add the copy doc and demo link to the QA section? It can be confusing in the "Screenshots" section, that section should be for Screenshots, right? 
And when I said a demo link, I meant a demo link directly to the page that the reviewer should review, in this case would be https://ubuntu-com-13888.demos.haus/ceph, since there's already a demo link, adding that would be just an additionally facility for people to review quicker.

File: templates/ceph/index.html (hunk 1 above)
Comment (immortalcodes) [reply templates/ceph/index.html]: Yeah, I have corrected that

**Pull request merged**